src/giphon/siphon.py

```diff
@@ -138,87 +140,91 @@ def siphon(
         transient=True,
     ) as progress:
         flat_tree_task_id = progress.add_task(
-            description="Looking for stuff to siphon...", total=None
+            description="Giphoning all the projects...", total=None
         )
 
         gl = get_gitlab_instance(url=gitlab_url, private_token=gitlab_token)
 
         groups = get_groups_from_path(namespace, gl)
 
-        flat_tree = []
+        all_project = []
 
-        for index, element in enumerate(
-            flatten_groups_tree(
-                groups=groups, gl=gl, archived=bool(clone_archived)
-            )
-        ):
-            progress.update(
-                flat_tree_task_id,
-                advance=1,
-                description=f"Found {index} elements...",
-            )
-            flat_tree.append(element)
+        for group in groups:
+            tmp_group = gl.groups.get(group.id)
+            project_in_group = tmp_group.projects.list(include_subgroups=True, all=True, archived=clone_archived)
```

Comment:
Using your method (without handling directly the `subgroups` themselves) won't download gitlab-ci variables for groups, right? confere siphon.py#227